library/core/src/marker.rs

```diff
@@ -809,6 +809,15 @@ pub trait Destruct {}
 #[cfg_attr(not(bootstrap), rustc_deny_explicit_impl)]
 pub trait Tuple {}
 
+/// A marker for things
+#[unstable(feature = "pointer_sized_trait", issue = "none")]
```

Comment:
Does it make sense to attach this to the dyn* implementation issue?

Reply:
I prefer to keep this as a perma-unstable trait until we're confident that this is the strategy we want to keep moving forward.

Reply:
Sounds good to me!